app/services/staff_action_logger.rb

```diff
@@ -830,4 +852,11 @@ def validate_category(category)
     raise Discourse::InvalidParameters.new(:category) unless category && category.is_a?(Category)
   end
 
+  def get_details(watched_word)
```

Comment:
```suggestion
  def get_watched_work_details(watched_word)
```

I think `get_details` is too generic considering the method is defined in the `staff_action_logger`. Otherwise, I think moving this method to the `WatchedWord` model might not be a bad idea.

Reply:
I agree. I've the method to `WatchedWord` model.